config/chime_science_run_gpu.yaml

```diff
@@ -348,9 +348,10 @@ gpu_beamform_hfb_output_buffers:
   gpu_beamform_hfb_output_buffer_3:
     kotekan_buffer: standard
 
+# Factor of 2 in the frame_size comes from the weights
 cpu_hfb_output_buffers:
   num_frames: buffer_depth
-  frame_size: num_data_sets * factor_upchan * num_frb_total_beams * sizeof_float
+  frame_size: num_data_sets * factor_upchan * num_frb_total_beams * 2 * sizeof_float
```

Comment:
Do the weights fully double the size of the data? (cc: @jrs65 )  That's quite a bit more storage than we'd estimated needing.

Reply:
For these buffers (from GPU -> accumulate stage) I don't think you need to increase the frame size.

As for the other buffers, I think for the `hfb` buffers you don't need any `frame_size` parameter at all as it's automatically calculated from the other config parameters.

Reply:
When I remove the `frame_size` from:
```
cpu_hfb_output_buffers:
    num_frames: buffer_depth
    #frame_size: num_data_sets * factor_upchan * num_frb_total_beams * sizeof_float
    metadata_pool: main_pool
    cpu_hfb_output_buffer_0:
        kotekan_buffer: standard
    beamform_hfb_output_buffer_0:
        metadata_pool: hfb_pool
        kotekan_buffer: standard
    beamform_hfb_output_buffer_read:
        metadata_pool: hfb_pool
        kotekan_buffer: standard
```
I get:
```
Failed to start kotekan with config file ../../config/tests/chime_frb_verification.yaml, error message: The config option: frame_size is required, but was not found in the path: /cpu_hfb_output_buffers/beamform_hfb_output_buffer_0
```

Reply:
I think that's because you are using the generic buffer (`kotekan_buffer: standard`)

Reply:
As I said, you'll need to keep it for the GPU->accumulate buffers as they are of type `standard`. For those of type `hfb` you should be able to remove it I think.